README.md

```diff
@@ -60,7 +61,9 @@
 My Nginx config file is shared [here](https://github.com/Parth-Vader/Scrapy-BenchCLI/blob/master/nginx.conf).
  Source : [How to install nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-16-04).
 
-The html files present in `sites.tar.gz` files were downloaded using a `scrapy spider` from the top sites from `Alexa top sites` list.
+The html files present in `sites.tar.gz` file were downloaded using a `scrapy spider` from the top sites from `Alexa top sites` list.
```

Comment:
Would it be possible to document how this was actually done? for example by providing the spider code.
It could be useful to re-build it or add sites later on.